python/kadalu_content_apis/objects.py

```diff
@@ -1,23 +1,23 @@
 import os
 from kadalu_content_apis.shares import Share
-from kadalu_content_apis.helpers import response_object_or_error
+from kadalu_content_apis.helpers import response_object_or_error, APIError
 
 class Document:
-    def __init__(self, conn, bucket_name, path):
+    def __init__(self, conn, folder_name, path):
         """ Intialise Document/Object """
         self.conn = conn
-        self.bucket_name = bucket_name
+        self.folder_name = folder_name
         self.path = path
 
 
     @classmethod
-    def create(cls, conn, bucket_name, path, data, object_type, immutable, version, lock, template):
-        """ Create object of both default("/") and with bucket-name """
+    def create(cls, conn, folder_name, path, data, object_type, immutable, version, lock, template):
+        """ Create object of both default("/") and with folder-name """
 
```

Comment:
`folder_name = folder_name.lstrip("/")` and then compare `if folder_name == ""`

Reply:
done